Spark/metrics.csv

```diff
@@ -0,0 +1,207 @@
+spark.stage.shuffle_write_bytes.median,gauge,,byte,,The median number of bytes written during a shuffle in the application's stages,0,spark,stg shfl wrt byt med
+spark.stage.shuffle_write_bytes.95percentile,gauge,,byte,,The 95th percentile number of bytes written during a shuffle in the application's stages,0,spark,stg shfl wrt byt 95
+spark.stage.shuffle_write_bytes.count,rate,,byte,,The number of times the number of bytes written during a shuffle in the application's stages was sampled,0,spark,stg shfl wrt byt cnt
+spark.stage.shuffle_write_records.max,gauge,,record,,The max number of records written during a shuffle in the application's stages,0,spark,stg shfl wrt rec mx
+spark.stage.shuffle_write_records.avg,gauge,,record,,The average number of records written during a shuffle in the application's stages,0,spark,stg shfl wrt rec avg
+spark.stage.shuffle_write_records.median,gauge,,record,,The median number of records written during a shuffle in the application's stages,0,spark,stg shfl wrt rec med
+spark.stage.shuffle_write_records.95percentile,gauge,,record,,The 95th percentile number of records written during a shuffle in the application's stages,0,spark,stg shfl wrt rec 95
+spark.stage.shuffle_write_records.count,rate,,record,,The number of times the number of records written during a shuffle in the application's stages was sampled,0,spark,stg shfl wrt rec cnt
+spark.stage.memory_bytes_spilled.max,gauge,,byte,,The max number of bytes in memory spilled in the application's stages,0,spark,stg mem byt spld mx
+spark.stage.memory_bytes_spilled.avg,gauge,,byte,,The average number of bytes in memory spilled in the application's stages,0,spark,stg mem byt spld avg
+spark.stage.memory_bytes_spilled.median,gauge,,byte,,The median number of bytes in memory spilled in the application's stages,0,spark,stg mem byt spld med
+spark.stage.memory_bytes_spilled.95percentile,gauge,,byte,,The 95th percentile number of bytes in memory spilled in the application's stages,0,spark,stg mem byt spld 95
+spark.stage.memory_bytes_spilled.count,rate,,byte,,The number of times the number of bytes in memory spilled in the application's stages was sampled,0,spark,stg mem byt spld cnt
+spark.stage.disk_bytes_spilled.max,gauge,,byte,,The max number of bytes in disk spilled in the application's stages,0,spark,stg dsk byt spld mx
+spark.stage.disk_bytes_spilled.avg,gauge,,byte,,The average number of bytes in disk spilled in the application's stages,0,spark,stg dsk byt spld avg
+spark.stage.disk_bytes_spilled.median,gauge,,byte,,The median number of bytes in disk spilled in the application's stages,0,spark,stg dsk byt spld med
+spark.stage.disk_bytes_spilled.95percentile,gauge,,byte,,The 95th percentile number of bytes in disk spilled in the application's stages,0,spark,stg dsk byt spld 95
+spark.stage.disk_bytes_spilled.count,rate,,byte,,The number of times the number of bytes in disk spilled in the application's stages was sampled,0,spark,stg dsk byt spld cnt

```

Comment:
for `spark.stage.disk_bytes_spilled` let's use "the number of bytes spilled to disk" instead of "the number of bytes in disk spilled"